Gopkg.toml

```diff
@@ -6,12 +6,18 @@
 required = [
     "git.fd.io/govpp.git/cmd/binapi-generator",
     "github.com/ungerik/pkgreflect",
-    "github.com/gogo/protobuf/protoc-gen-gogo"
+    "github.com/gogo/protobuf/protoc-gen-gogo",
+    "github.com/Shopify/sarama",
+    "github.com/bsm/sarama-cluster",
+    "github.com/coreos/etcd",
```

Comment:
Many of these don't need to be listed in `required`, since they are imported as direct or transitive dependency. Mainly only some programs which cannot be imported, but are used for code generation etc. should be here.